lib/Doctrine/DBAL/Driver/PDOSqlsrv/LastInsertId.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Driver\PDOSqlsrv;
+
+/**
+ * Last Id Data Container.
+ */
+class LastInsertId
+{
+    /** @var string */
+    private $id;
+
+    /**
+     * @param string $id
+     */
+    public function setId($id)
```

Comment:
Type declarations over docblocks: this is `string` and `: void`. Same for the rest of the patch overall.